charts/seed-bootstrap/charts/monitoring/templates/prometheus-vpa.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: "autoscaling.k8s.io/v1beta2"
+kind: VerticalPodAutoscaler
+metadata:
+  name: seed-prometheus-vpa
+  namespace: {{ .Release.Namespace }}
+spec:
+  targetRef:
+    apiVersion: {{ include "statefulsetversion" . }}
+    kind: StatefulSet
```

Comment:
Based on the experience with VPA so far, it is nice to specify `minAllowed` to avoid too aggressive scale downs.